components/04-views/dashboard/address-book/address-book--edit.hbs

```diff
@@ -0,0 +1,21 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            <div class="grid">
+                {{> '@dashboard-form' contactInfo }}
+
+                {{> '@dashboard-form' address }}
+            </div>
```

Comment:
the space between this columns is too small

Reply:
space added

Reply:
scpace too small between sidebar and main column
<img width="640" alt="screen shot 2017-09-07 at 12 07 28" src="https://user-images.githubusercontent.com/15080214/30158323-33521294-93c5-11e7-8318-a2edd3594235.png">

Reply:
added some more space

Reply:
but it looks weird when we have a table in content b/c space is added from 768px and not always it wrapping into 1/4 - 3/4. Sometimes it's waiting when table will fit to the content

Reply:
please add class with `flex-wrap: nowrap` for grid for 768 - 992px width. add also comment (scss) that this is fix for table width

Reply:
changed to better solution